src/runtime/qurt_hvx.cpp

```diff
@@ -57,7 +57,7 @@ WEAK_INLINE int _halide_prefetch_2d(const void *ptr, int width_bytes, int height
     return 0;
 }
 
-WEAK_INLINE int _halide_prefetch(const void *ptr, int size) {
+WEAK_INLINE int _halide_prefetch_1D(const void *ptr, int size) {
```

Comment:
1D -> 1d (the call site is 1d, and matches 2d convention)

Reply:
Done